tools/msvs/msi/i18n/de-de.wxl

```diff
@@ -9,8 +9,8 @@
     <String Id="MajorUpgrade_DowngradeErrorMessage">Eine neuere Version von [ProductName] ist bereits installiert. Der Installationsassistent wird jetzt geschlossen.</String>
 
     <!-- References like [ProductName] or $(var.ProductName) don't seem to work in Title attributes -->
-    <String Id="NodeRuntime_Title">io.js-Laufzeitumgebung</String>
-    <String Id="NodeRuntime_Description">Installation der [ProductName]-Laufzeitumgebung (iojs.exe).</String>
+    <String Id="NodeRuntime_Title">Node.js-Laufzeitumgebung</String>
+    <String Id="NodeRuntime_Description">Installation der [ProductName]-Laufzeitumgebung (node.exe).</String>
 
     <String Id="NodeAlias_Title">Verweis von node auf iojs</String>
```

Comment:
These two lines can be removed (no need for hardlinking iojs.exe to node.exe any longer)